index.js

```diff
@@ -67,6 +67,7 @@ const execCompose = (command, args, options) => new Promise((resolve, reject) =>
   });
 
   const result = {
+    exitCode: 0,
```

Comment:
I'd say `exitCode` should be `null` until process changes this to a specific value.

EDIT: This would also mean that `index.d.ts` file would need to reflect that with 
```typescript
interface IDockerComposeResult {
    exitCode: ?number;
    out: string;
    err: string;
}
```